### PR TITLE
Initial taxonomy api

### PR DESCRIPTION
**Pull request by gwarmstrong**
Here is a first draft of what the API spec for taxonomy might look like. Pretty similar to alpha. Lets a user figure out what resources are available for taxonomy, then will provide them with a summary of the taxonomies in those samples. The plan is to lean heavily on the existing taxonomy model.

### CODE REVIEW

**Comment by wasade** [comment]
Should this PR be against upstream?

File: microsetta_public_api/api/microsetta_public_api.yml [hunk 1]
```diff
@@ -205,6 +205,106 @@ paths:
           description: >
             Either `summary_statistics`, `return_raw`, or both are required to be true.
 
+  '/taxonomy/resources/available':
```
Comment (wasade) [inline microsetta_public_api/api/microsetta_public_api.yml]: is "resources" needed?

**Comment by gwarmstrong** [comment]
Ultimately yes. But it was branched off of the alpha statistics, so I compared it to that branch to highlight the changes. I'll merge it into that branch and open a PR against the upstream.

File: microsetta_public_api/api/microsetta_public_api.yml (hunk 1 above)
Comment (gwarmstrong) [reply microsetta_public_api/api/microsetta_public_api.yml]: Probably not, easy to change later, too.
```suggestion
  '/taxonomy/available':
```

**Pull request merged**